### PR TITLE
cplay-ng: init at 5.1.0

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [x] (Module updates) Added a release notes entry if the change is significant
  - [x] (Module addition) Added a release notes entry if adding a new NixOS module
  - [x] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/cplay-ng/default.nix [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      Instead of building an elaborate database of your music library, cplay
+      allows you to quickly browse the filesystem and enqueue files,
+      directories, and playlists.
+
+      The original cplay was started by Ulf Betlehem in 1998 and is no longer
+      maintained. This is a rewrite that aims to stay true to the original
+      design while evolving with a shifting environment.
+    '';
+    license = licenses.gpl2Plus;
+    platforms = platforms.all;
```
Comment (AndersonTorres) [inline pkgs/applications/audio/cplay-ng/default.nix:46]:
```suggestion
```

Python is not supported by "all" platforms.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Otherwise LGTM

File: pkgs/applications/audio/cplay-ng/default.nix (hunk 1 above)
Comment (fgaz) [reply pkgs/applications/audio/cplay-ng/default.nix:46]: Then python itself is going to block evaluation for this package too. cplay-ng itself does not have any particular platform restriction.

Comment (AndersonTorres) [reply pkgs/applications/audio/cplay-ng/default.nix:46]: I am not understanding your point. Python is the restriction per se.
And, as far as I remember, `meta.platforms` is automatically filled by `buildPythonApplication`. So, there is no need to set it except maybe for restrict it even more.

Comment (fgaz) [reply pkgs/applications/audio/cplay-ng/default.nix]: > meta.platforms is automatically filled by buildPythonApplication

oh right. i always forget this. removed.

**Pull request merged**